src/resources/app-en.json

```
@@ -1309,6 +1310,15 @@
     "placeholders": {
       "opt_in": "Text",
       "opt_out": "Text"
+    },
+    "chipLabels": {
+      "active": "Active",
+      "available": "Premium"
+    },
+    "modal": {
+      "title": "Request Health Record Data",
+      "content": "<0>Now it is possible to import participants' Electronic Health Care Data, and use that data with what you collect in your Mindlogger Activities.</0> <1>Please check out the website for more information.</1>",
+      "cta": "Go to website"
```

The design capitalizes "Website", and also "To", but I kinda think capitalizing "To" is unusual… so this my suggestion. Feel free to go with mine or with Figma!
```suggestion
      "cta": "Go to Website"
```

---

Updating 🤓